zipkin-ui/package.json

```diff
@@ -5,10 +5,10 @@
   "main": "index.js",
   "scripts": {
     "dev": "node node_modules/webpack-dev-server/bin/webpack-dev-server.js --hot --inline",
-    "test": "node node_modules/karma/bin/karma start --single-run",
+    "test": "node node_modules/karma/bin/karma start --single-run --optimize-minimize --optimize-occurrence-order --optimize-dedupe",
```

Comment:
I don't think karma understands these flags... anyways for unit tests we prefer fast runs over minimized code. (After all, unit test code is not shipped to the user)